libs/blocks/marquee/mnemonic-list.css

```diff
@@ -0,0 +1,33 @@
+/* Mnemonic list marquee */
+
+@media screen and (min-width: 800px) {
```

Comment:
was 800px provided in the requirements? If not, I propose to align this with merch cards, where tablet starts at `(min-width: 768px)`

Reply:
We should stick to our Main milo breakpoints. 768 OR 800 aren't part of the core milo or dexter media queries.

```
/* mobile only (rarely used - mobile first) */
@media (max-width: 600px)

/* tablet up */
@media screen and (min-width: 600px)

/* tablet large up (rarely used) */
@media (min-width: 900px)

/* desktop up */
@media screen and (min-width: 1200px)

/* desktop large up (rarely used) */
@media screen and (min-width: 1440px)
```

Reply:
Totally agree on this one, let's use consistent breakpoints

Reply:
Thank you all, I've changed to `min-width: 768px` as per Milo breakpoints.